TTYC.Constants/ConfigurationConstants.cs

```diff
@@ -3,5 +3,9 @@
 	public static class ConfigurationConstants
 	{
 		public const string DefaultConnection = "DefaultConnection";
+
```

Comment:
Let's make the security options some nested structure or at least mark them with #region